nextflow_schema.json

```diff
@@ -415,7 +579,28 @@
                     "default": "bwamem2",
                     "description": "Specifies the alignment algorithm to use - available options are 'bwamem2' and 'sentieon'.",
                     "fa_icon": "fas fa-align-center",
-                    "enum": ["bwamem2", "sentieon"]
+                    "enum": ["bwa", "bwamem2", "sentieon"]
+                },
+                "min_trimmed_length": {
+                    "type": "integer",
+                    "default": 40,
+                    "description": "Discard trimmed reads shorter than this",
+                    "help_text": "Minimum length of reads after adapter trimming. Shorter reads are discarded. The program default is 15 bp.  ",
+                    "fa_icon": "fas fa-less-than"
+                },
+                "mt_subsample_rd": {
+                    "type": "integer",
+                    "default": 150,
+                    "description": "Expected coverage to subsample mt alignment to.",
+                    "help_text": "To know more about this parameter check samtools' view documentation.",
```

Comment:
are you sure you mean "samtools view" here? If yes, this is rendered as markdown, so you can directly link to it here.